src/ScaleFactorMET.cc

```diff
@@ -119,9 +111,16 @@ float ScaleFactorMET::getMinThreshold()
   
 double ScaleFactorMET::getSF(double metnomu)
 {
-
   metnomu = mImposeBounds(metnomu);
-  return funcSF->Eval(metnomu);
+
+  double sf;
+  if (mPeriod != "2017" and metnomu == mRange.at(mPeriod).second) {
+	sf = 1.;
+  }
+  else {
+	sf = funcSF->Eval(metnomu);
```

Comment:
the functions in the SF files stop at 350 GeV right? doesn't this break for `metnomu>350`?

Reply:
it should be correct, see [this](https://github.com/bfonta/KLUBAnalysis/blob/700af10de76a5ee21bcaba4740d1c13ffd14a88c/src/ScaleFactorMET.cc#L114).